src/BackwardCompatiblityTests/BackwardCompatiblityTests.csproj

```diff
@@ -0,0 +1,90 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The project directory and file name are misspelled.

Reply:
Corrected

---
In reply to: [308738733](https://github.com/microsoft/axe-windows/pull/119#discussion_r308738733) [](ancestors = 308738733)